### DISCUSSION TITLE
Feature: Use correct region (locale) when displaying numbers

### DISCUSSION THREAD

**Discussion by namandixit** [victorbalssa/abacus#215] [Ideas]
Currently, the numbers (such as net worth) are printed without taking the proper locale into consideration, leading to the commas appearing according to the [Western Short Scale System](https://en.wikipedia.org/wiki/Long_and_short_scales) instead of the [Indic numbering system](https://en.wikipedia.org/wiki/Indian_numbering_system) on iPhones with Indian region settings.

To test this, go to _Settings_ - _General_ - _Language and Region_ - _Number Format_ and select _12,34,567.89_. The number formats in the app should change, but they don't.

**Comment by victorbalssa** [comment]
Hey @namandixit, does the version `0.11.1` fix this issue?
Thanks

**Reply by namandixit** [reply]
Yes, it does, thank you! Closing the thread.
